client/css/login.css

```diff
@@ -0,0 +1,51 @@
+body {
+    background-image: url(../assets/images/Couple_01.jpeg);
+    background-repeat: no-repeat;
+    background-color: black;
+    background-size: cover;
+}
+
+#logo_container {
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-self: center;
+    text-align: center;
+}
+
+#logo {
+    display: block;
+    margin-top: 0px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    -webkit-filter: drop-shadow(5px 5px 5px #222);
+    filter: drop-shadow(5px 5px 5px #222);
+
+}
+
+#login-container {
+    display: grid;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+    background: lightyellow;
+    : 100px;
```

Comment:
you're missing some attribute here @DouglasMacKrell. all i see is ": 100px;" nothing on the left of the colon